io.github.dweymouth.supersonic.yml

```diff
@@ -0,0 +1,209 @@
+  # https://docs.flatpak.org/en/latest/electron.html for how it's done
+  # with electron apps. alternatively, mpv could be a "shared module",
+  # see:
+  # https://docs.flatpak.org/en/latest/manifests.html#shared-modules
+  # https://github.com/flathub/shared-modules
+  - name: mpv
+    buildsystem: meson
+    build-options:
+      cflags: -DNDEBUG=1
+    config-opts:
+      - -Dbuild-date=false
+      - -Dlibmpv=true
+      - -Dmanpage-build=disabled
+      - -Dalsa=enabled
+      - -Dcdda=disabled
+      - -Ddvbin=disabled
+      - -Ddvdnav=disabled
+      - -Djavascript=disabled
+      - -Dlibarchive=disabled
+      - -Dlua=disabled
+      - -Dpulse=enabled
+      - -Dsdl2=disabled
+      - -Dshaderc=disabled
+      - -Dvulkan=disabled
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+    sources:
+      - type: archive
+        url: https://github.com/mpv-player/mpv/archive/refs/tags/v0.35.1.tar.gz
+        sha256: 41df981b7b84e33a2ef4478aaf81d6f4f5c8b9cd2c0d337ac142fc20b387d1a9
+        x-checker-data:
+          type: anitya
+          project-id: 5348
+          stable-only: true
+          url-template: https://github.com/mpv-player/mpv/archive/refs/tags/v$version.tar.gz
+    modules:
+    - name: libass
+      config-opts:
+        - --disable-static
+      sources:
+        - type: archive
+          url: https://github.com/libass/libass/releases/download/0.17.1/libass-0.17.1.tar.xz
+          sha256: f0da0bbfba476c16ae3e1cfd862256d30915911f7abaa1b16ce62ee653192784
+          x-checker-data:
+            type: anitya
+            project-id: 1560
+            stable-only: true
+            url-template: https://github.com/libass/libass/releases/download/$version/libass-$version.tar.xz
+        - type: script
+          commands:
+            - autoreconf -fiv
+          dest-filename: autogen.sh
+      cleanup:
+        - /include
+        - /lib/pkgconfig
+        - /share
+    - name: ffmpeg
+      cleanup:
+        - /include
+        - /lib/pkgconfig
+        - /share/ffmpeg/examples
+      config-opts:
+        # this is almost like io.mpv.Mpv, except we removed:
+        # --enable-libbs2b
+        # --enable-libjxl
+        # --enable-libmysofa
+        # --enable-libx264
+        # --enable-libx265
+        # because those would require yet another dependency tree
+        - --disable-debug
+        - --disable-doc
+        - --disable-programs
+        - --disable-static
+        - --enable-encoder=png
+        - --enable-gnutls
+        - --enable-gpl
+        - --enable-shared
+        - --enable-version3
+        - --enable-libaom
+        - --enable-libass
+        - --enable-libdav1d
+        - --enable-libfreetype
+        - --enable-libmp3lame
+        - --enable-libopus
+        - --enable-libtheora
+        - --enable-libv4l2
+        - --enable-libvorbis
+        - --enable-libvpx
+      sources:
+        - type: git
+          url: https://github.com/FFmpeg/FFmpeg.git
+          mirror-urls:
```

Comment:
`(flatpak-builder:2): flatpak-builder-WARNING **: 20:11:25.885: Unknown property mirror-urls for type BuilderSourceGit`

Reply:
odd. that's from: 

https://github.com/flathub/com.icanblink.blink/blob/75a76c08017c41833a30dc879f00c1aec9662a28/modules/io.mpv.Mpv.json#L57

Reply:
they don't read the logs and we don't make these fatal errors.